### PR TITLE
Improve the sparse docs on python renderer

### PR DESCRIPTION
**Pull request by oz123**
### What does this PR do?

Added documentation
### What issues does this PR fix or reference?

 Probable fix for #36963

### CODE REVIEW

File: salt/renderers/py.py [hunk 1]
```diff
@@ -68,7 +94,7 @@ def run():
             ],
         }
 
-        return config
+         return config
```
Comment (babilen) [inline salt/renderers/py.py]: Whitespace change necessary/correct ?

File: salt/renderers/py.py [hunk 2]
```diff
@@ -54,7 +80,7 @@ def run():
         }
 
         config[home + '/.s3cfg'] = {
-            'file.managed': [
+             'file.managed': [
```
Comment (babilen) [inline salt/renderers/py.py]: Is this whitespace change correct/intentional?

**Review by wwentland — COMMENTED** [review]
Check whitespace changes

**Comment by cachedout** [comment]
@babilen Because there was no response from the OP I edited the PR myself to remove the whitespace. It looks to me like you are correct.

**Comment by babilen** [comment]
@cachedout Thank you. I wasn't entirely sure, but it looked off which is why I asked.

**Pull request merged**